examples/session_view.cpp

```diff
@@ -102,18 +102,18 @@ void session_view::render()
 		, esc("48;5;238")
 #endif
 		, esc("1")
-		, add_suffix(m_cnt[0][m_wasted_bytes_idx]).c_str()
-		, color(add_suffix(upload_rate, "/s"), col_red).c_str()
-		, color(add_suffix(m_cnt[0][m_sent_payload_idx]), col_red).c_str()
-		, color(to_string(m_cnt[0][m_queued_reads_idx], 3), col_red).c_str()
-		, color(to_string(m_cnt[0][m_queued_writes_idx], 3), col_green).c_str()
+		, add_suffix(float(m_cnt[0][m_wasted_bytes_idx])).c_str()
+		, color(add_suffix(float(upload_rate), "/s"), col_red).c_str()
+		, color(add_suffix(float(m_cnt[0][m_sent_payload_idx])), col_red).c_str()
+		, color(to_string(int(m_cnt[0][m_queued_reads_idx]), 3), col_red).c_str()
```

Comment:
and I wonder if it would make sense to have an overload that takes an int64_t (which I assume this conversion does, right?)